components/uniset/src/uniset.rs

```diff
@@ -213,18 +233,19 @@ impl UnicodeSet {
     /// in a single `O(1)` step
     ///
     /// Example: `contains_range(&0, &10)`
-    pub fn contains_range(&self, start: u32, end: u32) -> Result<bool, Box<dyn Error>> {
+    pub fn contains_range(&self, start: char, end: char) -> Result<bool, Box<dyn Error>> {
```

Comment:
Idea: change the function signature to

```rust
pub fn contains_range(&self, range: &dyn RangeBounds<char>) -> bool
```

and then you can call `range.start_bound()` and `range.end_bound()`.  The call site would be,

```rust
my_uni_set.contains_range(&('a'..='z'));
```

One argument is nicer than two, especially if it is a trait, because then someone else can implement that trait, and you can pass their object directly to `contains_range`.

Reply:
Also, look at file:///home/zbraniecki/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/share/doc/rust/html/std/ops/enum.Bound.html and consider aligning the API with Bound, `BTreeMap::range` etc.

Reply:
agreed and changed